APE18.rst

```diff
@@ -0,0 +1,53 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+APE 10 previously determined supported CPython versions, and provided a list up to the 4.0 release.
+Since APE 10 was written the wider scientific Python community have written and largely adopted `NEP 29`_ as a policy for supported versions of both CPython and Numpy.
```

Comment:
While *I'm* fine with this, I think it might be good to give at least one concrete example of a way the more "accelerated" schedule will help things. Like what py3.7 features would we want to implement sooner rather than later? (manylinux2010 might be the answer! Or data classes?)

Reply:
I added a small justification in the backwards compatibility section, I don't really want to do a "let's adopt 3.7 because new features things" because I would rather this stand the test of time. I am happy to pull the justification section up here if you think it's a better place for it?

Reply:
I see your point on "stand the test of time", but I was thinking the point of this is to give a couple of concrete examples to what is currently a very abstract phrasing.  So just meant something like "For example, at the time of the py3.7 release infrastructure maitainers found their life would be easier if manylinux2010 was an option". Or similar.

But if you really want it to be timeless, I'm ok with either pulling it up here or leaving as-is :shrug:.